examples/nextjs/tsconfig.json

```diff
@@ -1,19 +1,21 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES2017",
```

Comment:
With full knowledge of my js+react ignorance, I'm not sure about how I feel what this could potentially mean to someone checking the example. Can we use something newer than 7 years old or is this the way it is?

Reply:
I mean es5 i even older than that. Typically you set the target here to be the oldest version you are trying to support. Also like the other ones this comes stock out of the nextjs example app

Reply:
This is for the code that runs on the browser, so it should be as old as the oldest browser supported.